src/config/__mocks__/database.ts

```diff
@@ -0,0 +1,40 @@
+import { execSync } from 'child_process';
+import { join } from 'path';
+
+import { PrismaClient } from '@prisma/client';
+import { v4 } from 'uuid';
+
+const generateDatabaseURL = (schema: string) => {
+  if (!process.env.POSTGRES_DB_CONNECTION_URL) {
+    throw new Error('please provide a database url');
```

Comment:
Minor Developer Experience improvemebt suggestion (oen should have clear error messages):

```suggestion
    throw new Error('`POSTGRES_DB_CONNECTION_URL` has not been defined');
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/72/commits/dd79cd103a872db6a10e4e4e104e624980c6d91a